lib/src/covid19client.dart

```diff
@@ -51,29 +51,27 @@ class Covid19Client {
   /// It returns a list of [Covid19CountryAllStatus].
   ///
   /// The [status] can be: 'confirmed', 'recovered' or 'deaths'.
-  Future<List<Covid19CountryAllStatus>> getDayOne(
+  Future<List> getDayOne(
```

Comment:
Could you please specify the type of the elements in the list?
```dart
Future<List<Covid19CountryEx>> getDayOne(
```

Reply:
aye, I'm on it!